pyproject.toml

```diff
@@ -32,6 +32,7 @@ test = [
     "lightning-thunder; python_version >= '3.10'",
 ]
 all = [
+    "galore-torch",              # galore
```

Comment:
Should we vendor what we need out of this package instead? It currently requires transformers which is excessive for an optimizer: https://github.com/jiaweizzhao/GaLore/blob/master/galore_torch/adamw.py#L10

A precedent is that we have our own LoRA impl instead of installing Microsoft's

Reply:
I agree, let's do that.